apps/passport-client/components/screens/ProveScreen/SemaphoreGroupProveScreen.tsx

```diff
@@ -164,3 +175,14 @@ async function fillArgs(
 const LineWrap = styled.div`
   margin-bottom: 16px;
 `;
+
+const ErrorContainer = styled.div`
+  border: 2px solid var(--danger);
+  color: white;
+  border-radius: 8px;
+  padding: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--bg-lite-primary);
```

Comment:
i recommend making this more like the danger button, so `background: var(--danger);`

+ then you don't need a separate border

Reply:
OK - updated screenshot in the PR description